src/content/developers/tutorials/Using-UseDApp-with-unity/index.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Using UseDApp with Unity
+description: Learn how to use UseDApp to interact with Unity engine project 
+author: "lucasespinosa28"
+tags: ["usedapp", "unity","react","dapp"]
+skill: intermediate
+lang: en
+sidebar: true
+sourceUrl: https://github.com/lucasespinosa28/usedappwithUnity
+---
+## Introduction {#introduction}
+In this tutorial, you will learn how to use [UseDApp](https://usedapp.readthedocs.io/en/latest/) a React plugins for rapid Dapp development, 
+and how to interact with a Unity using react, you will get your account balance inside a Unity and send a transaction from unity.
+
+### You need have [Unity](https://unity.com/) installed 
+
+## Unity project {#unity-project}
+On the Unity will have a text to receive the account balance, and a button to send a transaction using UseDApp,
```

Comment:
_Inline_ code snippets should be wrapped in single backticks instead of triple (triple for code blocks).

Check the grammar on this as well... not fully clear what the instructions here are